code/controllers/subsystem/vote.dm

```diff
@@ -379,4 +396,10 @@ SUBSYSTEM_DEF(vote)
 	else if(owner.ckey)
 		var/datum/player_details/P = GLOB.player_details[owner.ckey]
 		if(P)
-			P.player_actions -= src
\ No newline at end of file
+			P.player_actions -= src
+
+#undef PEACE2
+#undef PEACE1
+#undef BALANCED
+#undef CHAOS1
+#undef CHAOS2

```

Comment:
Some of these don't exist anymore

Reply:
fixed in https://github.com/Citadel-Station-13/Citadel-Station-13/pull/9557/commits/85fd53d61595a0bc5ce0dce467b89278b8678341